glean-core/csharp/Glean/ManualPings.cs

```diff
@@ -0,0 +1,82 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using Mozilla.Glean.Private;
+using System;
+
+namespace Mozilla.Glean.GleanPings
+{
+    internal sealed class GleanInternalPingsOuter
```

Comment:
By looking at the [Koltin generated code](https://dpaste.org/RP2o), we don't need/want ping definitions to be lazily initialized.